detekt-rules/src/test/kotlin/io/gitlab/arturbosch/detekt/rules/style/UseRequireSpec.kt

```diff
@@ -83,6 +90,15 @@ class UseRequireSpec : Spek({
             assertThat(subject.lint(code)).isEmpty()
         }
 
+        it("does not report an issue if the exception thrown has a non-String argument") {
+            val code = """
+                fun test(throwable: Throwable) {
+                    if (throwable !is NumberFormatException) throw IllegalArgumentException(throwable)
+                }
+            """.trimIndent()
```

Comment:
It's not necessary to call `.trimIndent()` again, since `lint()` already does it for us.

```suggestion
            """
```